CurrentTimeApp/CurrentTimeApp.csproj

```diff
@@ -8,7 +8,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.9" />
+		<PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="7.0.0" />
```

Comment:
This is currently failing for the following error:

```
Run dotnet restore --locked-mode
  dotnet restore --locked-mode
  shell: /usr/bin/bash -e {0}
  env:
    DOTNET_ROOT: /home/runner/.dotnet
  Determining projects to restore...
/home/runner/work/CurrentTimeApp/CurrentTimeApp/CurrentTimeApp/CurrentTimeApp.csproj : error NU1004: The package reference Microsoft.AspNetCore.Components.WebAssembly version has changed from [6.0.9, ) to [7.0.0, ).The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file. [/home/runner/work/CurrentTimeApp/CurrentTimeApp/CurrentTimeApp.sln]
  Failed to restore /home/runner/work/CurrentTimeApp/CurrentTimeApp/CurrentTimeApp/CurrentTimeApp.csproj (in 1[3](https://github.com/jjliggett/CurrentTimeApp/actions/runs/3421106796/jobs/5696796755#step:6:3)[7](https://github.com/jjliggett/CurrentTimeApp/actions/runs/3421106796/jobs/5696796755#step:6:8) ms).
Error: Process completed with exit code 1.
```

Additionally, this update requires an update to the .NET version. Microsoft.AspNetCore.Components.WebAssembly v7.0.0 is not compatible with net6.0.